src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -603,6 +583,9 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
                                    cAlphaFieldNames(2),
                                    cAlphaArgs(2)));
             ErrorsFound = true;
+        } else {
+            state.dataHeatBal->ZoneAirBalance(Loop).ZonePtr = zoneNum;
+            state.dataHeatBal->Zone(zoneNum).zoneOABalanceIndex = Loop;
```

Comment:
Seems like you could have just added the else zoneOABalanceIndex = Loop and been done with it with no local int zoneNum.